parm/config/config.base.emc.dyn

```diff
@@ -227,11 +233,77 @@ export lobsdiag_forenkf=".true."
 export DO_GLDAS="YES"
 export gldas_cyc=00
 
-# run wave component
-export DO_WAVE="NO"
+# APP settings
+export APP="ATM"
+
+if [ $APP = "ATM" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+elif [ $APP = "ATMW" ]; then
+  export DO_WAVE="YES"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+elif [ $APP = "S2S" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="YES"
+  export DO_CICE="YES"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+elif [ $APP = "S2SW" ]; then
+  export DO_WAVE="YES"
+  export DO_MOM6="YES"
+  export DO_CICE="YES"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+elif [ $APP = "ATMAERO" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="YES"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+fi
+
 if [[ "$SDATE" -lt "2019020100" ]]; then DO_WAVE="NO" ; fi # no rtofs in GDA
 export WAVE_CDUMP="both"
 
+# Coupling settings
+export cpl=".true."
+export cplflx=".true."
+if [ $DO_CICE = "YES" ]; then
+  export cplice=".true."
+else
+  export cplice=".false."
+fi
+if [ $DO_GOCART = "YES" ]; then
+  export cplgocart=".true."
+else
+  export cplgocart=".false."
+fi
+if [ $DO_WAVE = "YES" ]; then
+  export cplwav=".true."
+  export cplwav2atm=".true."
```

Comment:
cplwav2atm is not a "DO_WAVE" yes/no option, we can have DO_WAVE= true but cplwave2atm false

Reply:
cplwav2atm right now is more appropriately set by APP=ATMW (more/less GFSv16), then cplwav2atm=false, but if APP=S2SW then cplwav2atm=true.

Reply:
Removed that line, it will default to .false. in config.base now. Thanks!